scripts/01946-Dante.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$myData = "<h2>My name is David Dante Frank and my HNG Id is HNG-01946 and I am using PHP for stage 2 task</h2>";
```

Comment:
Please include a space and your email after the word `task` above as explained [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)